src/components/layout/CompanyClaim/CompanyClaim.module.scss

```diff
@@ -54,13 +54,14 @@
         background-color: $header-bg;
         display: flex;
         align-items: center;
-        justify-content: center;
+        justify-content: space-between;
         font-size: 14px;
-        color: rgb(224, 227, 237);
+        color: $background-white;
         position: absolute;
         top: 50%;
         right: 0;
         transform: translate(50%, -50%);
+        padding-left: $base-padding*0.3;
```

Comment:
dobrze jakby środek liczy zawsze był na prawej krawędzi ikony

Reply:
Zmienione, zmieniłem lekko wielkość elementu koszyka, dodałem background które podąża za wielkością liczby aby lepiej to wyglądało.